plugins/vpp/l3plugin/descriptor/route.go

```diff
@@ -21,10 +21,10 @@ import (
 	"strings"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/pkg/errors"
-
 	"github.com/ligato/cn-infra/logging"
 	"github.com/ligato/cn-infra/utils/addrs"
+	"github.com/pkg/errors"
```

Comment:
but why did you move it to group with cn-infra packages?

Reply:
I'm using `goimports` and I've got Go imports setup in Goland in 3 groups: std, other modules, current module.

Reply:
oh, thanks, sorry for bothering :)

Reply:
It's no bother ;)